docs/writing-stories/index.md

```diff
@@ -405,7 +405,7 @@ In such cases, it makes sense to render a different function for each story:
 
 <!-- prettier-ignore-end -->
 
-You can also reuse stories from the child `ListItem` in your `List` component. That’s easier to maintain because you don’t have to keep the identical story definitions updated in multiple places.
+You can also reuse story definitions from the child `ListItem` in your `List` component. That’s easier to maintain because you don’t have to keep the identical details updated in multiple places.
```

Comment:
```suggestion
You can also reuse _story data_ from the child `ListItem` in your `List` component. That’s easier to maintain because you don’t have to keep it updated in multiple places.
```

What do you think?